examples/llama2/llama2_qlora.json

```diff
@@ -2,9 +2,40 @@
     "input_model": {
         "type": "PyTorchModel",
         "config": {
-            "model_script": "user_script.py",
-            "io_config": "get_merged_decoder_with_past_io_config",
-            "dummy_inputs_func": "get_merged_decoder_with_past_dummy_inputs",
+            "io_config": {
+                "input_names": [
+                    "input_ids",
+                    "attention_mask"
+                ],
+                "output_names": [
+                    "logits"
+                ],
+                "input_shapes": [
+                    [
```

Comment:
nitpick: Would it make sense to collapse certain key values like this one on a single line. Sometime this gives novice user an impression that they need to provide complicated input.

Reply:
I used json formatter to get the `complicated` one.

Just updated with collapsed json.